website/static/js/script.js

```diff
@@ -11,8 +11,19 @@
     return resp.json();
   }
 
-  function uniqify(array, predicate) {
-    return array.reduce((acc, curr) => acc.find(a => predicate(a) === predicate(curr)) ? acc : acc.push(curr) && acc, []);
+  function uniqueBySlug(array) {
+    var predicate = function (o) {
+      return o.fromAccount.slug;
+    }
+    return array.reduce(function(acc, curr) {
+      return acc.find(function (a) {
+        return predicate(a) === predicate(curr);
+      }) ? acc : acc.push(curr) && acc
+    }, []);
+  }
+
+  function sortByTotalDonations(a, b) {
```

Comment:
This function needs to return a number, not Boolean.

Reply:
True, it's not the first time I'm doing that mistake :-/